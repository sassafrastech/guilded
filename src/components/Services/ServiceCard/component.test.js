import React from 'react';
import renderer from 'react-test-renderer';

import Component from './component';

it('renders', () => {
  const tree = renderer.create(
    <Component icon="Icon" title="Title" description="Description" />,
  );
  expect(tree.toJSON()).toMatchSnapshot();
});
