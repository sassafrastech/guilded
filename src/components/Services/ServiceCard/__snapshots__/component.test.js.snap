// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders 1`] = `
<div
  className="makeStyles-container-1"
>
  <img
    alt=""
    className="makeStyles-icon-2"
    src="Icon"
  />
  <h6
    className="MuiTypography-root makeStyles-title-3 MuiTypography-subtitle1"
  >
    Title
  </h6>
  <p
    className="MuiTypography-root MuiTypography-body2"
  >
    Description
  </p>
</div>
`;
